index.html

```diff
@@ -1,32 +1,31 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="style.css" />
-    <link rel="preconnect" href="https://fonts.googleapis.com" />
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap" rel="stylesheet"/>
-    <title>Portfolio</title>
-  </head>
-  <body>
-   
-  <main class="container">
-    <section class="home">
-    <header class="mobile-nav">
-        <h2 class="header-logo"><a href=""> My Logo </a></h2>
-        <nav class="first-nav">
-          <div class="hamburger"><img src="./images/Union.svg" alt="un" /></div>
-        </nav>
-    </header>
 
-    </section>
-    <section>
-       <h1 class="heading_second">
-         I'm Silvia<br />
-         Glad to see <br />
-         you!
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="index.css" />
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap" rel="stylesheet" />
+  <title>Portfolio</title>
+</head>
+
+<body>
+  <main class="container">
+    <div class="backg-img">
+      <section class="home_page">
+        <header class="mobile-nav">
```

Comment:
- [ ] [Optional] I strongly recommend you to read [this article](https://www.w3schools.com/html/html5_semantic_elements.asp) about semantic tags, it would help you to understand how to structure your HTML documents because currently, it has some mistakes. After reading it, I strongly suggest you to make some improvements in this project